crm-project/src/main/java/CRM/entity/Attribute.java

```diff
@@ -25,4 +26,11 @@ public class Attribute {
     private String name;
     private String description;
 
```

Comment:
Also, please add @ToString function to this function please.
I know it's not your code, but if you use it, it'll be good if you added it :)